src/SFA.DAS.FAT.Web/Views/Accessibility/AccessibilityStatement.cshtml

```diff
@@ -0,0 +1,78 @@
+﻿@using Microsoft.AspNetCore.Routing
+@using SFA.DAS.FAT.Web.Infrastructure
+
+
+<div class="govuk-width-container">
+    <a asp-route="@RouteNames.ServiceStart" class="govuk-back-link">Back</a>
+
+    <main class="govuk-main-wrapper">
+
```

Comment:
Are all the links meant to redirect you away? or should they open separate tabs/windows